ClientTests/UserAgentTests.swift

```diff
@@ -43,7 +43,7 @@ class UserAgentTests: XCTestCase {
         XCTAssertTrue(mockUserDefaults.synchronizeCalled)
         XCTAssertNotNil(mockUserDefaults.registerValue)
         XCTAssertNotNil(mockUserDefaults.string(forKey: "LastFocusVersionNumber"))
-        XCTAssertTrue(((mockUserDefaults.registerValue!["UserAgent"] as? String)?.contains(AppInfo.config.productName))!)
+        XCTAssertTrue(((mockUserDefaults.registerValue!["UserAgent"] as? String)?.contains("FxiOS"))!)
```

Comment:
@sblatz, I believe this test failure was addressed in commit https://github.com/mozilla-mobile/focus-ios/commit/63c513478e0b6e371cb829e449a1a1e113e83251